src/view/gui/fMainWindow.cpp

```diff
@@ -964,6 +963,8 @@ fMainWindow::fMainWindow()
   m_tabWidget->setTabText(m_tabWidget->indexOf(tumorPanel), QApplication::translate("fMainWindow", "Seed Points", 0));
   m_tabWidget->setTabText(m_tabWidget->indexOf(drawingPanel), QApplication::translate("fMainWindow", "Drawing", 0));
   m_tabWidget->setTabText(m_tabWidget->indexOf(imagesPanel), QApplication::translate("fMainWindow", "Images", 0));
+
+  preferenceDialog = new PreferencesDialog(nullptr);
```

Comment:
Please add a  brief comment as to why this was moved to the end for future reference